examples/semver.peggy

```diff
@@ -39,13 +33,17 @@ preReleaseIdentifier
 
 buildIdentifier
   = alphanumericIdentifier
-  / digit+
+  / $digit+ // Not a number, buildIdentifiers aren't semantically significant.
 
+// If there is a non-digit anywhere, this label is alphanumeric, and
+// is compared lexically.  Return a string.
 alphanumericIdentifier
-  = digit* nonDigit identifierChar*
+  = $(digit* nonDigit identifierChar*)
 
+// Any semantically significant numbers are turned into integers for
+// later numeric comparison.
 numericIdentifier
-  = '0' / (positiveDigit digit*)
+  = n:('0' / $(positiveDigit digit*)) { return parseInt(n, 10); }
```

Comment:
Formally should parse be `BigInt(n)` instead of `parseInt(n, 10)`.

Reply:
eh, I guess.  We don't have to worry about IE anymore.